src/engines/html5/html5.js

```diff
@@ -109,6 +132,14 @@ export default class Html5 extends FakeEventTarget implements IEngine {
   }
 
   /**
+   * Gets the result from the canAutoPlay test.
+   * @returns {Promise<boolean>} - The can auto play result.
+   */
+  canAutoPlay(): Promise<boolean> {
+    return Html5.canAutoPlay();
```

Comment:
why do you need instance method as well as static?

Reply:
Because I don't know which engine will be chosen at run time. We can alternativly access this._engine.constructor.canAutoPlay() but I think it's ugly. Any suggestion?